liquidity-sources/src/u256_decimal.rs

```diff
@@ -0,0 +1,97 @@
+use primitive_types::U256;
+use serde::{de, Deserializer, Serializer};
+use serde_with::{DeserializeAs, SerializeAs};
+use std::fmt;
+
+pub struct DecimalU256;
+
+impl<'de> DeserializeAs<'de, U256> for DecimalU256 {
+    fn deserialize_as<D>(deserializer: D) -> Result<U256, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        deserialize(deserializer)
+    }
+}
+
+impl SerializeAs<U256> for DecimalU256 {
+    fn serialize_as<S>(source: &U256, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serialize(source, serializer)
+    }
+}
+
+pub fn serialize<S>(value: &U256, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    serializer.serialize_str(&value.to_string())
+}
+
+pub fn deserialize<'de, D>(deserializer: D) -> Result<U256, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    struct Visitor {}
+    impl<'de> de::Visitor<'de> for Visitor {
+        type Value = U256;
+
+        fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+            write!(formatter, "a u256 encoded as a decimal encoded string")
+        }
+
+        fn visit_str<E>(self, s: &str) -> Result<Self::Value, E>
+        where
+            E: de::Error,
+        {
+            U256::from_dec_str(s).map_err(|err| {
+                de::Error::custom(format!("failed to decode {:?} as decimal u256: {}", s, err))
+            })
+        }
+    }
+
+    deserializer.deserialize_str(Visitor {})
+}
+
+/// Converts an amount of units of an ERC20 token with the specified amount of
+/// decimals into its decimal representation as a string.
+///
+/// # Examples
+///
+/// ```rust
+/// use model::u256_decimal::format_units;
+///
+/// assert_eq!(format_units(42u64.into(), 0), "42");
+/// assert_eq!(format_units(1_337_000u64.into(), 6), "1.337000")
+/// ```
+pub fn format_units(amount: U256, decimals: usize) -> String {
+    let str_amount = amount.to_string();
```

Comment:
Let's remove this is if it's not being used.

Reply:
Yes, removed.